code/clement/html_css_flask/lab05/templates/index.html

```diff
@@ -0,0 +1,111 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="{{url_for('static', filename='normalize.css')}}">
+
+    <!-- css -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+
+    <!-- ----------Bootstrap Bundle-------- -->
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p"
+        crossorigin="anonymous"></script>
+
+    <link rel="stylesheet" href="{{url_for('static', filename='styles.css')}}">
+
+    <title>Burrito Order Form</title>
+
+</head>
+
+<body>
+    <div class="container">
+        <h1>Burrito Order Form</h1>
+        <div class="row">
+            <div class="col-12 col-lg-6">
+                <img src="https://food.fnr.sndimg.com/content/dam/images/food/fullset/2009/12/13/0/FNM_011009-W-N-Dinners-004_s4x3.jpg.rend.hgtvcom.406.305.suffix/1371591444720.jpeg"
+                    class="img-thumbnail mt-5" alt="...">
```

Comment:
The alt tag for the image should be utilized to assist people who need assistance. It allows the browser to say what is rather than something unrecognizable.